packages/pangea-sdk/pangea/services/audit/models.py

```diff
@@ -373,7 +373,24 @@ class SearchEvent(APIResponseModel):
     signature_verification: EventVerification = EventVerification.NONE
 
 
-class SearchOutput(PangeaResponseResult):
+class SearchResultOutput(PangeaResponseResult):
+    """
+    Return class after a pagination search
+
+    Arguments:
+    count -- The total number of results that were returned by the search.
+    events -- A list of matching audit records.
+    root -- A root of a Merkle Tree.
+    unpublished_root -- Root of a unpublished Merkle Tree
+    """
+
+    count: int
+    events: List[SearchEvent]
+    root: Optional[Root] = None
+    unpublished_root: Optional[Root] = None
```

Comment:
This is how to format class properties:

```suggestion
    count: int
    """The total number of results that were returned by the search."""
    
    events: List[SearchEvent]
    """A list of matching audit records."""
    
    root: Optional[Root] = None
    """A root of a Merkle Tree."""
    
    unpublished_root: Optional[Root] = None
    """Root of a unpublished Merkle Tree."""
```

EDIT: oh, this just copy-paste of below. Well, consider this a nice-to-have then.

Reply:
Oh, okey, maybe we should refactor all python docs